eeps/eep-0009.md

```diff
@@ -435,45 +433,47 @@ Open questions:
   should be activated
 - handling of Encodings.
 
-        matches(BinOrString, RegExp)-> Return
-        matches(BinOrString, RegExp, {StartIndex, EndIndex})-> Return
+matches(BinOrString, RegExp)-> Replacementeturn
+matches(BinOrString, RegExp, {StartIndex, EndIndex})-> Return
 
-        BinOrString = binary() | string()
-        RegExp = string() | MatchSpec
-        MatchSpec = tuple() as returned by match_compile/1
-        StartIndex = EndIndex = integer()
+    BinOrString = binary() | string()
+    RegExp = string() | MatchSpec
+    MatchSpec = tuple() as returned by match_compile/1
+    StartIndex = EndIndex = integer()
+
+    Return = 0 | [ {Start, Length, [CapturedPatterns]} ]
 
-        Return = 0 | [ {Start, Length, [CapturedPatterns]} ]
+Finds all matches of the regular expression RegExp in BinOrString.
 
-  Finds all matches of the regular expression RegExp in BinOrString.
+Example:
 
-  Example:
+    > binary:regex_matches(<<"aaa">>, "a").
+    [{1,1,[]},{2,1,[]},{3,1,[]}]
 
-        > binary:regex_matches(<<"aaa">>, "a").
-        [{1,1,[]},{2,1,[]},{3,1,[]}]
+<!-- nothing to see here -->
```

Comment:
This also separates code blocks, in Markdown. I didn't use the previous "hack" since this is not indented (inside a list item).